app/controllers/admins_controller.rb

```diff
@@ -13,6 +13,13 @@ def approve
         @item.save
         redirect_to admins_path
     end
+    
+    def approve_p
+        @procedure=Procedure.find(params[:id])
+        @procedure.status="approved"
```

Comment:
when there is no string interpolation use single strings.

Reply:
Changed " to ' in all files for this pull request where appropriate.